api/src/quest/application/index.js

```diff
@@ -12,6 +12,25 @@ const ERRORS = {
 
 const register = async function (server) {
   server.route([
+    {
+      method: 'GET',
+      path: '/api/combined-courses',
+      config: {
+        auth: false,
+        handler: questController.getByCode,
+        validate: {
+          query: Joi.object({
+            filter: Joi.object({
+              code: Joi.string()
+                .regex(/[^a-zA-Z0-9]/, { invert: true })
```

Comment:
```suggestion
                .regex(/[a-zA-Z0-9]/)
```

Il faudrait aussi faire la même chose sur campaign-details-route.

A retester !

Reply:
`/^[a-zA-Z0-9].*$/` convient mieux je pense :)